src/ol/source/tilevectorsource.js

```diff
@@ -299,9 +305,14 @@ ol.source.TileVector.prototype.loadFeatures =
             tileUrlFunction(urlTileCoord, 1, projection);
         if (goog.isDef(url)) {
           tiles[tileKey] = [];
-          var loader = ol.featureloader.loadFeaturesXhr(url, this.format_,
-              goog.partial(success, tileKey));
-          loader.call(this, extent, resolution, projection);
+          var tileSuccess = goog.partial(success, tileKey);
+          if (goog.isDef(this.tileVectorLoadFunction_)) {
+            this.tileVectorLoadFunction_.call(this, url, tileSuccess);
```

Comment:
Is there a reason to execute this with the source as `this` keyword? I don't think it is necessary.